src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -76,6 +76,19 @@ export class UpperBarManager {
     return icons.sort((a, b) => iconsOrder[a.displayName] - iconsOrder[b.displayName]);
   }
 
+  public focusPluginButton(pluginId: number) {
+    let pluginButton;
+    const controls = this.getControls(this.iconsOrder);
+    const pluginElement = controls.find((control) => control.id === pluginId)?.componentRef?.current?.base as HTMLElement;
+    if (pluginElement) {
+      pluginButton = pluginElement.querySelector('[tabindex="0"]') as HTMLElement;
+    } else {
+      const rightControls = this.displayedBarComponentRefs.Playback.current?.base as HTMLElement;
+      const buttons = rightControls.querySelectorAll('[tabindex="0"]');
```

Comment:
I would do `querySelector('[tabindex]:not([tabindex="-1"])')` but that's not critical